django_spanner/features.py

```diff
@@ -1026,12 +1027,20 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             "db_functions.math.test_ceil.CeilTests.test_null",  # noqa
             "db_functions.math.test_ceil.CeilTests.test_transform",  # noqa
             "db_functions.math.test_cos.CosTests.test_null",  # noqa
+            "db_functions.math.test_cos.CosTests.test_transform",  # noqa
             "db_functions.math.test_cot.CotTests.test_null",  # noqa
+            "db_functions.math.test_degrees.DegreesTests.test_decimal",  # noqa
```

Comment:
It looks like some tests (like this one) fail because spanner numeric fields really aren't precise enough. Computed server-side, [spanner's "degrees"](https://github.com/vi3k6i5/python-spanner-django/blob/ce68f3efb1f1cb3b48222ef4cfa0f79312616fa5/django_spanner/functions.py#L185) isn't almostEqual to `Decimal(math.degrees(...))`. But what if we changed the precision in the [decimal context](https://docs.python.org/3/library/decimal.html#context-objects)?

I don't see an obvious place to change the context, but it does look like we should be setting the precision in the context to match [spanner's numeric scale limit](http://cloud/spanner/docs/data-types#numeric_type) before doing some numeric ops.

Maybe a problem for another PR, but I thought I'd mention it here since this one adds some new skipped tests because of the precision mismatch.

Reply:
Opening an investigation issue for the same: https://github.com/googleapis/python-spanner-django/issues/627